### ISSUE TITLE
EPERM errors when building ejected Android project on Windows

### ISSUE THREAD

**Issue by esamelson** [expo/expo-cli#240]
We have gotten a few reports of an error that prevents an ejected project from building properly in Android Studio on Windows. There seems to be some sort of open file handle or permissions error that prevents some intermediate files from being deleted properly, and this causes the build to fail. These reports may or may not be related but show similar symptoms. More details here: https://forums.expo.io/t/run-the-app-in-android-emulator-after-ejecting-gives-eperm-operation-not-permitted-lstat/16178 and here: https://blog.expo.io/expokit-might-not-be-so-scary-3c26769215fa (search for "Almost every time I stop and start").

**Comment by lLenn** [comment]
I've had the same issue and none of the provided solutions worked for me. I found that the issue is that metro is watching the android files and throwing an error whenever Android Studio is manipulation the files. I've gotten around this issue by adding a metro.config.js file and exluding the android folder. I posted my solution on the expo forums: [link](https://forums.expo.io/t/error-eperm-operation-not-permitted-lstat/19221)
The regular expression I'm using is what I've found metro uses to initialise + .*\\android\\.*
This may cause some other android directories to be exluded, but this was a quick fix for me (I tried add __dirname but this gave some errors and I was tired so I gave up)
I feel using the metro config is the correct way to deal with this as it has no business in the android directory.
